### PR TITLE
Kindly review New Changes to Dev Branch.

### PR DESCRIPTION
**Pull request by Yash-VIT**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
-node_modules
+
```
Comment (ibmpariwesh) [inline .gitignore:11]: don't remove

File: backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java [hunk 2]
```diff
@@ -25,20 +25,21 @@
 	@Autowired
 	SectorService sectorService;
 	
-	@PutMapping("/create")
+	@PostMapping("/create")
```
Comment (ibmpariwesh) [inline backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java]: /sector

File: backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java [hunk 3]
```diff
@@ -25,20 +25,21 @@
...
 	}
 	
 	@GetMapping("/display")
 	@ApiOperation(value = "Get All Sector Value")
-	public ResponseEntity<List<SectorDto>> findSector(){
+	public ResponseEntity<Iterable<SectorDto>> findSector(){
 		return  ResponseEntity.ok(sectorService.findSector());
 	}
 	
 	@GetMapping("/display/{id}")
```
Comment (ibmpariwesh) [inline backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java]: /sector/{id}

File: backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java [hunk 4]
```diff
@@ -47,5 +48,11 @@ public ResponseEntity findSectorById(@PathVariable int id){
 		
 		return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Sector with id "+ id+" not found.");
 	}
+	
+	@DeleteMapping("/delete/{id}")
```
Comment (ibmpariwesh) [inline backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java]: /sector/{id}

**Review by ibmpariwesh — CHANGES_REQUESTED** [review]
Please rectify

**Review by sankalpjain99 — APPROVED** [review]
Looks good

File: .gitignore (hunk 1 above)
Comment (Yash-VIT) [reply .gitignore:11]: Added

File: backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java (hunk 2 above)
Comment (Yash-VIT) [reply backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java]: modified

File: backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java (hunk 3 above)
Comment (Yash-VIT) [reply backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java]: Modified

File: backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java (hunk 4 above)
Comment (Yash-VIT) [reply backend/sector/src/main/java/com/sgtrain/sector/controller/SectorController.java]: Modified

**Pull request merged**